.github/workflows/publish.yml

```
@@ -1,31 +1,31 @@
 name: Publish to PyPI
+  on:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 2: mapping values are not allowed in this context

(syntax-check)